addon/helpers/string-to-camel-case.js

```diff
@@ -0,0 +1,21 @@
+import { helper } from '@ember/component/helper';
+
+/**
+ * camelCase a string.
+ *
+ * @public
+ * @function stringToCamelCase
+ * @param {string} value The string to camelCase.
+ * @returns {string} The camelCased string.
+ */
+export function stringToCamelCase([value]) {
```

Comment:
Perhaps 
```js
export function stringToCamelCase([value = '']) {
```
to guard against npes.

Applies to all other helpers.

Reply:
Good idea ... sensible defaults added throughout codebase.